homeassistant/components/blink/services.py

```diff
@@ -86,23 +89,30 @@ async def async_handle_save_video_service(call: ServiceCall) -> None:
         video_path = call.data[CONF_FILENAME]
         if not hass.config.is_allowed_path(video_path):
             _LOGGER.error("Can't write %s, no access to path!", video_path)
```

Comment:
We don't have to log and raise

```suggestion
```

Reply:
Makes sense once I actually think about it :)